src/components/Calculator.js

```diff
@@ -1,80 +1,103 @@
+  });
 
-    <div>
-      <div className="button-rows">
-        <button type="button" name="AC">
-          AC
-        </button>
-        <button type="button" name="+/-">
-          +/-
-        </button>
-        <button type="button" name="%">
-          %
-        </button>
-        <button type="button" name="/">
-          ÷
-        </button>
-      </div>
-      <div className="button-rows">
-        <button type="button" name="7">
-          7
-        </button>
-        <button type="button" name="8">
-          8
-        </button>
-        <button type="button" name="9">
-          9
-        </button>
-        <button type="button" name="x">
-          X
-        </button>
-      </div>
+  const onClickEvent = (event) => {
+    const buttonName = event.currentTarget.textContent;
+    const newState = calculate(calcObj, buttonName);
+    setCalculation(newState);
+  };
 
-      <div className="button-rows">
-        <button type="button" name="4">
-          4
-        </button>
-        <button type="button" name="5">
-          5
-        </button>
-        <button type="button" name="6">
-          6
-        </button>
-        <button type="button" name="-">
-          -
-        </button>
-      </div>
-      <div className="button-rows">
-        <button type="button" name="1">
-          1
-        </button>
-        <button type="button" name="2">
-          2
-        </button>
-        <button type="button" name="3">
-          3
-        </button>
-        <button type="button" name="+">
-          +
-        </button>
+  const { next, operation, total } = calcObj;
+
+  return (
+    <div className="calc-container">
+      <div className="output-bar">
+        { total }
+        {' '}
+        { operation }
+        {' '}
+        { next }
       </div>
-      <div className="last-row">
-        <button type="button" name="0">
-          0
-        </button>
-        <button type="button" name=".">
-          .
-        </button>
-        <button type="button" name="=">
-          =
-        </button>
+
+      <div>
+        <div className="button-rows">
+          <button type="button" name="AC" onClick={onClickEvent}>
+            AC
+          </button>
+          <button type="button" name="+/-" onClick={onClickEvent}>
+            +/-
+          </button>
+          <button type="button" name="%" onClick={onClickEvent}>
+            %
+          </button>
+          <button type="button" name="/" onClick={onClickEvent}>
+            ÷
+          </button>
+        </div>
+        <div className="button-rows">
+          <button type="button" name="7" onClick={onClickEvent}>
+            7
+          </button>
+          <button type="button" name="8" onClick={onClickEvent}>
+            8
+          </button>
+          <button type="button" name="9" onClick={onClickEvent}>
+            9
+          </button>
+          <button type="button" name="x" onClick={onClickEvent}>
+            X
+          </button>
```

Comment:
- [ ] @cherelemma kindly check on your multiplication function am getting the error below when trying to multiply 2 numbers
![image](https://user-images.githubusercontent.com/61408860/227703693-4a8a220a-00c0-4926-aa6c-c0793cbabe32.png)